packages/forklift-console-plugin/src/modules/Providers/views/migrate/PlansCreateForm.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { ProviderModelGroupVersionKind } from '@kubev2v/types';
+import { ResourceLink } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Divider,
+  Form,
+  FormGroup,
+  List,
+  ListItem,
+  Panel,
+  PanelMain,
+  PanelMainBody,
+  TextInput,
+} from '@patternfly/react-core';
+
+import { PageAction, setPlanDescription, setPlanName, setPlanTargetNamespace } from './actions';
+import { CreateVmMigrationPageState } from './reducer';
+
+export const PlansCreateForm = ({
+  state: { newPlan: plan, validation },
+  dispatch,
+}: {
+  state: CreateVmMigrationPageState;
+  dispatch: (action: PageAction<unknown, unknown>) => void;
+}) => {
+  const { t } = useForkliftTranslation();
+  return (
+    <Form isWidthLimited className="forklift-section-secret-edit">
+      <FormGroup label={t('Plan name')} isRequired fieldId="planName" validated={validation.name}>
+        <TextInput
+          isRequired
+          type="text"
+          id="planName"
+          value={plan?.metadata?.name}
+          validated={validation.name}
+          onChange={(value) => dispatch(setPlanName(value?.trim() ?? '', []))}
+        />
+      </FormGroup>
+      <FormGroup label={t('Description')} fieldId="description">
+        <TextInput
+          type="text"
+          id="description"
+          value={plan?.spec?.description}
+          onChange={(value) => dispatch(setPlanDescription(value))}
+        />
+      </FormGroup>
+      <Divider />
+      <FormGroup label={t('Source provider')} fieldId="sourceProvider">
+        <span id="sourceProvider">
+          <ResourceLink
+            name={plan?.spec?.provider?.source?.name}
+            namespace={plan?.spec?.provider?.source?.namespace}
+            groupVersionKind={ProviderModelGroupVersionKind}
+          />
+        </span>
+      </FormGroup>
+      <FormGroup label={t('Selected VMs')} fieldId="selectedVms">
```

Comment:
it's just "[N] vms selected", you will only show the individual names if the user press a "show vms" link/key/icon

Reply:
in this PR only the number is displayed - the details view requires more attention